draft-ietf-taps-transport-security.md

```diff
@@ -418,7 +419,9 @@ like TCP.
 
 Application Payload Security Protocols:
 
-- SRTP {{?RFC7201}}
```

Comment:
This deletes the only reference to RFC 7201 from our document.
Is it intentional to drop this reference and use RFC4571 instead here?